qodana.yaml

```
@@ -0,0 +1,17 @@
+version: "1.0"
+linter: jetbrains/qodana-clang:2024.2-eap
+bootstrap:  sh ./prepare_cordana.sh
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)